src/App.tsx

```diff
@@ -1,15 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [searchQuery, setSearchQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    return movie.title.toLowerCase().includes(searchQuery.toLowerCase())
+      || movie.description.toLowerCase().includes(searchQuery.toLowerCase());
```

Comment:
i think it is better to do the lowercase method one time instead two ( setSearchQuery(blablabla.toLowerCase()) )

Reply:
also user can missclick and put two or more spaces after input text and there will be no result((